plugins/polio/models.py

```diff
@@ -174,12 +176,31 @@ class RoundDateHistoryEntry(models.Model):
     previous_ended_at = models.DateField(null=True, blank=True)
     started_at = models.DateField(null=True, blank=True)
     ended_at = models.DateField(null=True, blank=True)
+    # Deprecated.
     reason = models.CharField(null=True, blank=True, choices=DelayReasons.choices, max_length=200)
+    reason_for_delay = models.ForeignKey(
+        "ReasonForDelay", on_delete=models.PROTECT, null=True, blank=True, related_name="round_history_entries"
+    )
     round = models.ForeignKey("Round", on_delete=models.CASCADE, related_name="datelogs", null=True, blank=True)
     modified_by = models.ForeignKey("auth.User", on_delete=models.PROTECT, null=True, blank=True)
     created_at = models.DateTimeField(auto_now_add=True)
 
 
+class ReasonForDelay(SoftDeletableModel):
+    name = TranslatedField(models.CharField(_("name"), max_length=200), {"fr": {"blank": True, "null": True}})
```

Comment:
```suggestion
    name = TranslatedField(models.CharField(_("name"), max_length=200), {"fr": {"blank": True}})
```

It's better to avoid using [`null` on string-based fields](https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.Field.null).